### ISSUE TITLE
how to set optional parameters for slim.batch_norm

### ISSUE THREAD

**Issue by patienceFromZhou** [davidsandberg/facenet#527]
here is my batch_norm_params, which is soon fed into normalizer_params.
![image](https://user-images.githubusercontent.com/31264567/32766637-d25474bc-c94a-11e7-967f-14e46ccee6a9.png)

however, when i print tf.trainable_variables, there are only mean, variance and beta for BN, missing gamma..
![image](https://user-images.githubusercontent.com/31264567/32766817-816296f0-c94b-11e7-92da-210169a038cd.png)

how to change the default settings? such as adding gamma or simply reserve mean and variance.
